src/js/labels/labels.js

```diff
@@ -16,6 +16,7 @@ export const verbs = {
   PAUSE: () => 'pause',
   MUTE: () => 'mute',
   UNMUTE: () => 'unmute',
+  SELECT_AIRPLAY: () => 'select air play',
```

Comment:
In the text itself, is it more accurate to say "select airplay device"?

Reply:
Not sure tbh. One of the options is to "reselect" playback in the browser. If you prefer "select air play device" though, I think it still communicates effectively enough. Just lmk.

Reply:
Based on the other comment, let's move this to just `AIRPLAY`.

Reply:
cool. I'm going to make it "air play" as it's consumed by screen readers and I don't want the possibility of the robots getting confused about pronunciation. 😄

Reply:
will also rename the function `AIRPLAY` (realized that's what you meant).

Reply:
👍